public/docs/_examples/quickstart/ts/app/app.module.ts

```diff
@@ -1,13 +1,16 @@
 // #docregion
-import { NgModule }      from '@angular/core';
-import { BrowserModule } from '@angular/platform-browser';
+import { Component, NgModule } from '@angular/core';
+import { BrowserModule }       from '@angular/platform-browser';
 
-import { AppComponent }   from './app.component';
+@Component({
+  template: `<h1>Hello Angular!</h1>`,
+  selector: 'my-app'
+})
+export class AppComponent { }
```

Comment:
I think removing `export` is simpler. With it included, I am looking to see where it is imported. This is not a building block to start an app with, right? It's intended to show the simplest path to make A2 work. If so, less is better.